mobile/lib/features/student_management/presentation/widgets/drawer_overlay.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+
```

Comment:
Doesn't the drawer overlay already exist?

Reply:
it does, 2 older versions, one in user and other in class, but i made a better version for student user case, also needs to be onde file so i'll just delete the older ones and put it as a widget outside those user cases as well